test/e2e/utils/utils.go

```diff
@@ -128,6 +129,23 @@ func SetupProwConfig(resourceType string) (project, serviceAccount string) {
 	// [PROJECT_NUMBER]-compute@developer.gserviceaccount.com
 	serviceAccount = fmt.Sprintf("%v-compute@developer.gserviceaccount.com", resp.ProjectNumber)
 	glog.Infof("Using project %v and service account %v", project, serviceAccount)
+
+	// Set Cloud KMS permissions on compute service account
+	// TODO: Use the API to make this call instead of exec-ing gcloud command
+	computeSystemSA := fmt.Sprintf("service-%v@compute-system.iam.gserviceaccount.com", resp.ProjectNumber)
```

Comment:
i'm not sure if this is the proper way we should be adding iam roles to project wide system accounts

Reply:
agreed, looking into alternatives now